src/hooks/apiCall.js

```diff
@@ -2,16 +2,21 @@ import axios from "axios"
 
 const apiUrl = 'https://nc-news-b2gg.onrender.com/api/'
 
+const apiReq = axios.create({
+    baseURL: apiUrl,
+    timeout: 1000,
+})
+
 export const get = (endpoint, queries) => {
```

Comment:
not sure how useful this is for you as the get/post/patch methods already exist on axios - you are just adding an additional wrapper around these methods here!